src/cls/rgw/cls_rgw_types.h

```diff
@@ -199,12 +199,14 @@ struct rgw_bucket_dir_entry_meta {
   std::string owner_display_name;
   std::string content_type;
   uint64_t accounted_size;
+  uint64_t size_rounded;
```

Comment:
This needs to be guarded in #ifdef WITH_RADOSGW_MOTR

Reply:
can not guard it since we are initializing the variable in the constructor initialization list.
In the case of rados as a backend the variable will have value=0 and it will never be encoded or decoded. Does this behaviour look fine?

Reply:
@sumedhak27 : I don't think we should be changing Ceph's core structure/class w/out Ceph's prior permission. If we want such functionality, we could always define our own class/struct and extend from their type.